include/seqan/bam_io/read_bam.h

```diff
@@ -267,7 +267,7 @@ readRecord(BamAlignmentRecord & record,
     TQualIter qitEnd = end(record.qual, Standard());
     for (TQualIter qit = begin(record.qual, Standard()); qit != qitEnd;)
         *qit++ = '!' + *it++;
-    if (!empty(record.qual) && (record.qual[0] - '!') == '\xff')
+    if (!empty(record.qual) && char(record.qual[0] - '!') == '\xff')
```

Comment:
Is `static_cast` not working here?

Reply:
It works, but it is more verbose and less explicit IMHO.